docs/console/design-your-projects/custom-resources.md

```diff
@@ -0,0 +1,128 @@
+---
+id: custom-resources
+title: Create Custom Resources
+sidebar_label: Create Custom Resources
+---
+
+## What is a Custom Resource
+
+A Custom Resource allows you to define custom objects that are not part of the standard Console supported resources.
+
+For example, it is possible to:
+
+- manage some Kubernetes Custom Resource managed by the cluster, e.g. the Traefik `IngressRoute`;
+- with [External Configuration Generator](/console/company-configuration/providers/extensions/orchestrator-generator/overview.mdx), generate templates for the other orchestrators.
+
+## How to manage a Custom Resource
+
+It is possible to manage the Custom Resource from inside the Design section of the Console.
+
+### Create a Custom Resource
+
+It is possible to create a new Custom Resource from Marketplace or from scratch.
+
+#### Create a Custom Resource from Marketplace
+
+To create a resource from Marketplace, you need to select the Custom Resource you want to create.
+
+At the moment, the only supported Custom Resources to create are the ones inside the private Company marketplace.  
+To allow the user to create a Custom Resource from marketplace, you need to apply it using `miactl` ([see here for details](#publish-a-custom-resource-inside-the-marketplace)).
+
+Once click on create from Marketplace, you will see the marketplace items you can use inside your project.
+
+![Create from Marketplace](./img/custom-resources/create-from-marketplace.png)
+
+#### Create a Custom Resource from Scratch
+
+To create a resource from scratch, you need to provide the following information:
+
+- **name**: the name of the Custom Resource. It will be used to generate the manifest, and cannot be changed during updates;
+- **apiVersion**: the apiVersion of the Custom Resource, it can be any string. If you are creating a Kubernetes Custom Resource, it must be the apiVersion of the Custom Resource Definition;
+- **kind**: the kind of the Custom Resource, it can be any string. If you are creating a Kubernetes Custom Resource, it must be the kind of the Custom Resource Definition.
+
+In creation, you will see the preview of the generated manifest.
+
+![Create from scratch](./img/custom-resources/create-from-scratch.png)
+
+### Update a Custom Resource
+
+Once created a Custom Resource, it is possible to update it. Click on the Custom Resource you want to update in the sidebar, and you will see a YAML editor.
```

Comment:
```suggestion
Once you have created a Custom Resource, it will be possible to update it. Click on the Custom Resource you want to update in the sidebar, and you will see a YAML editor.
```